share/cmake/modules/Findpystring.cmake

```diff
@@ -77,6 +77,10 @@ endif()
 if(NOT pystring_FOUND)
     include(ExternalProject)
 
+    if(APPLE)
+        set(CMAKE_OSX_DEPLOYMENT_TARGET ${CMAKE_OSX_DEPLOYMENT_TARGET})
+    endif()
+
```

Comment:
May I ask what's the intention behind this set of change ? This should've been appended to `pystring_CMAKE_ARGS` so that pystring is built for the same minimum macOS version as OCIO. 
Same applies to other find modules where the same patterns is applied.